packages/clarity/clarity.0.4.0/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+version: "0.4.0"
+maintainer: "joyprophecy@gmail.com"
+author: "joyprophecy@gmail.com"
+homepage: "https://github.com/IndiscriminateCoding/clarity"
+bug-reports: "https://github.com/IndiscriminateCoding/clarity/issues"
+dev-repo: "https://github.com/IndiscriminateCoding/clarity.git"
+license: "BSD3"
+available: [ ocaml-version >= "4.04.0" ]
+depends: [ "dune" {build} ]
+build: [
+  [ "dune" "subst" ] {pinned}
```

Comment:
Does this need `"-n" name` as well?

Reply:
As I understand it takes "name" value from dune-project file.